tools/release

```diff
@@ -45,44 +45,45 @@ print()
 
 # Perform local backup, go to tools dir to run it.
 cd(tooldir)
-sh('./make_tarball.py')
-sh('mv ipython-*.tgz %s' % ipbackupdir)
 
-# Build release files
-sh('./build_release %s' % ipdir)
+if 'upload' in sys.argv:
```

Comment:
Unsure it is worth it. We will likely forget to update common variables if we split it.